static/index.html

```diff
@@ -1,104 +1,106 @@
-<!--
-Copyright 2019 Google LLC.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
--->
-<!doctype html>
-<html>
+<!DOCTYPE html>
+<html lang="en">
   <head>
-    <title>Google App Engine Flexible Environment - Golang Websockets Chat</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta charset="utf-8">
+    <title>Foxtrot</title>
+    <link
+      rel="icon"
+      href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🦊</text></svg>"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
https://stackoverflow.com/questions/19506028/do-you-need-to-close-meta-and-link-tags-in-html says to not use the slash in the tag for meta and link tags for HTML5 documents.

Reply:
Prettier does that and it seems it can't be turned off:
https://github.com/prettier/prettier/issues/5246

sigh.
I'd prefer to leave it seeing as the SO article also says it does no harm.

Reply:
of course, I could just not use prettier, but it still does a lot of goodness for the rest...

Reply:
prettier making it uglier. sigh. It does a little harm - it leaves noise in the source. but never mind.

Reply:
I kind of assumed it might be prettier doing it, but it was not clear that you had run prettier - wasn't mentioned anywhere. I guess it's automatic in your editor?